### PR TITLE
Update check isAuthenticated

### PR DESCRIPTION
**Pull request by trunges21**
# Explain your changes

_Suppose there is a related issue with enough detail for a reviewer to understand your changes fully. In that case, you can omit an explanation and instead include either “Fixes #XX” or “Updates #XX” where “XX” is the issue number._

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

### CODE REVIEW

File: dist/SDK/KindeSDK.d.ts [hunk 1]
```diff
@@ -89,15 +89,12 @@ declare class KindeSDK extends runtime.BaseAPI {
...
-     * `TokenResponse`. It represents the token that needs to be refreshed. If this parameter is not
-     * provided, the function will try to retrieve the token from the storage using the
-     * `Storage.getToken()` method.
+     * @param {string} [refreshToken] - The refresh token value.
      * @returns The `useRefreshToken` function is returning the result of calling the `fetchToken`
      * function with a `FormData` object containing the necessary parameters for refreshing an access
      * token.
      */
-    useRefreshToken(token?: TokenResponse | null): Promise<TokenResponse>;
+    useRefreshToken(refreshToken: string): Promise<TokenResponse>;
```
Comment (rairaman) [inline dist/SDK/KindeSDK.d.ts:97]: @trunges21 Is this change backwards compatible for existing users?

**Review by rairaman — COMMENTED** [review]
@trunges21 Just left the 1 question there. Looks good otherwise.

File: dist/SDK/KindeSDK.d.ts (hunk 1 above)
Comment (trunges21) [reply dist/SDK/KindeSDK.d.ts:97]: This is just a private function. It is not made public to users

Comment (rairaman) [reply dist/SDK/KindeSDK.d.ts:97]: Ok, thanks.

**Pull request merged**